packages/config/lib/options.ts

```diff
@@ -0,0 +1,524 @@
+const validate = require('./validation')
```

Comment:
This file shouldn't be added. in 10.0-release branch, this content is under src/options.ts. Should be a small update to that file.